ccd-definition/CaseEvent.json

```diff
@@ -470,6 +470,21 @@
     "ShowSummary": "Y",
     "EndButtonLabel": "Save and continue"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "uploadC2",
+    "Name": "Upload a C2",
+    "Description": "Upload a c2 to the case",
+    "DisplayOrder": 3,
```

Comment:
If display order 3 please but it in appropriate place.

Reply:
Have moved down to bottom of the list for now (displayOrder: 10). Checking with business wether this should appear perhaps closer to documents.

Reply:
Caught up with business. Upload the C2 should appear after the documents option in both submitted and in gatekeeping.